### ISSUE TITLE
bug mat-list-option: ScreenReader(JAWS) reads not selected for checked mat-list-option

### ISSUE THREAD

**Issue by nikhil4326** [angular/components#19458]
#### Reproduction

Use StackBlitz to reproduce your issue: [https://stackblitz.com/edit/angular-mat-options?file=app%2Flist-selection-example.html](https://stackblitz.com/edit/angular-mat-options?file=app%2Flist-selection-example.html)

 
Steps to reproduce:
1. Use mat-selection-list along with mat-list-option to show the checkbox list.
2. Check and disable the mat-list-option using selected and disabled attribute(pre-populate with checked and disabled).
3. Use JAWS screen reader to the values of label of the mat-option-list
4. Jaws reads checked and disabled items as 'Not selected {{label Name}}'.
 
 
#### Expected Behavior

What behavior were you expecting to see?
Screen reader should read label name or 'selected {{label name}}' when mat-option-list is disabled and checked.

#### Actual Behavior

What behavior did you actually see?
JAWS screen reader reading it as 'not selected {{label Name}}' even mat-option-list is checked. This behavior is only when mat-option-list is disabled. 

#### Environment

- Angular: 7.2.16
- CDK/Material: 7.3.7

**Comment by jelbourn** [comment]
We won't be able to test this in JAWS until we're back in the office, but my hunch is that this is a JAWS bug if we're correctly setting `aria-selected` and `aria-disabled`.

**Comment by nikhil4326** [comment]
Any updates?

**Comment by vinayshuklag** [comment]
> #### Reproduction
> Use StackBlitz to reproduce your issue: https://stackblitz.com/edit/angular-mat-options?file=app%2Flist-selection-example.html
> 
> Steps to reproduce:
> 
> 1. Use mat-selection-list along with mat-list-option to show the checkbox list.
> 2. Check and disable the mat-list-option using selected and disabled attribute(pre-populate with checked and disabled).
> 3. Use JAWS screen reader to the values of label of the mat-option-list
> 4. Jaws reads checked and disabled items as 'Not selected {{label Name}}'.
> 
> #### Expected Behavior
> What behavior were you expecting to see?
> Screen reader should read label name or 'selected {{label name}}' when mat-option-list is disabled and checked.
> 
> #### Actual Behavior
> What behavior did you actually see?
> JAWS screen reader reading it as 'not selected {{label Name}}' even mat-option-list is checked. This behavior is only when mat-option-list is disabled.
> 
> #### Environment
> * Angular: 7.2.16
> * CDK/Material: 7.3.7

Hi Team, did we get any update on this? I am also facing the same issue

**Comment by vipinc38** [comment]
We are also experiencing the same issue with mat-list-option as mentioned in the description above. Can i have an update on this thread?

**Comment by nikhil4326** [comment]
Any resolution or workaround for this issue.

**Comment by zarend** [comment]
I cannot reproduce. Does the issue persist for you? We may have already fixed this.

Note that sometimes JAWS announces the not-checked options as "not selected" but it doesn't announce the checked options as "selected". That seems to be the way JAWS works.

**Comment by zarend** [comment]
Hello @nikhil4326 ,

Thank you for reporting this issue. Unfortunately we are not able to take action if we cannot reproduce it. This might have already been fixed. If the issue persists, please provide reproduction steps or file a new issue report.

- Zach
